services/web/server/src/simcore_service_webserver/projects/projects_db.py

```diff
@@ -345,48 +347,52 @@ async def __load_projects(
 
     async def _get_project(
         self,
+        connection: SAConnection,
         user_id: int,
         project_uuid: str,
-        exclude_foreign: Optional[List] = None,
+        exclude_foreign: Optional[List[str]] = None,
```

Comment:
can we have TESTS of this variants, please?
do you think this approach (i.e. these options) can be generalized to any database resource?

Reply:
added a new test that was completely missing of project_db... with some new tests but not all of them yet.